tests/plots/test_decision.py

```diff
@@ -15,7 +17,108 @@ def test_random_decision(random_seed):
         rs.standard_normal(size=(20, 5)),
         show=False
     )
-    pl.close()
+    plt.close()
+
+@pytest.mark.mpl_image_compare
+def test_decision_plot():
+    # base_value,
+    # shap_values,
+    fig = plt.figure()
+
+    X, y = shap.datasets.adult(n_points=10)
+    rfc = sklearn.ensemble.RandomForestClassifier()
+    rfc.fit(X, y)
+    ex = shap.TreeExplainer(rfc)
+    result_values = ex(X)
+    shap.decision_plot(result_values.base_values[0, 1],
+                       result_values.values[:, :, 1],
+                       show=False,
+                       return_objects=True,
+                       title="Decision Plot",
+                       link="identity")
+    plt.tight_layout()
+    return fig
+
+@pytest.mark.mpl_image_compare
+def test_decision_plot_single_instance():
+    # base_value,
+    # shap_values,
+    fig = plt.figure()
+
+    X, y = shap.datasets.adult(n_points=10)
+    rfc = sklearn.ensemble.RandomForestClassifier()
+    rfc.fit(X, y)
+    ex = shap.TreeExplainer(rfc)
+    result_values = ex(X)
+    shap.decision_plot(result_values.base_values[0, 1],
+                       result_values.values[0, :, 1],
+                       features=X.iloc[0],
+                       show=False,
+                       new_base_value=0,
+                       return_objects=True,
+                       )
+    plt.tight_layout()
+    return fig
+
+
+@pytest.mark.mpl_image_compare
+def test_decision_plot_interactions():
+    fig = plt.figure()
+
+    X, y = shap.datasets.adult(n_points=10)
+    rfc = sklearn.ensemble.RandomForestClassifier()
+    rfc.fit(X, y)
+    ex = shap.TreeExplainer(rfc)
+    result_values = ex(X, interactions=True)
+    shap.decision_plot(result_values.base_values[0, 1],
+                       result_values.values[:, :, :, 1],
+                       features=X,
+                       show=False,
+                       )
+    plt.tight_layout()
+    return fig
+
+@pytest.fixture
+def values_features():
```

Comment:
Can this fixture also be used in the tests above?

Reply:
done